src/main/java/edu/hm/hafner/analysis/parser/IdeaInspectionParser.java

```diff
@@ -43,11 +43,13 @@ private Report parseProblems(final List<Element> elements) {
                 if (problemClass.isPresent()) {
                     Element problem = problemClass.get();
                     issueBuilder.setFileName(stripPathPrefix(file))
-                            .setLineStart(Integer.parseInt(getChildValue(element, "line")))
                             .setCategory(StringEscapeUtils.unescapeXml(getValue(problem)))
                             .setMessage(StringEscapeUtils.unescapeXml(getChildValue(element, "description")))
                             .setModuleName(StringEscapeUtils.unescapeXml(getChildValue(element, "module")))
                             .setSeverity(getPriority(problem.getAttribute("severity")));
+                        if(!getChildValue(element, "line").equals("-")){
```

Comment:
Extract the value into a local variable and reuse later

Reply:
Maybe it would be much simpler when you use `IntegerParser.parseInt`, then everything should work out of the box.

Reply:
Thanks for the feedback. As recommended I change the parser to IntegerParser.